cola/views.py

```diff
@@ -50,13 +43,14 @@ def get(self, request: HttpRequest, **kwargs: dict) -> HttpResponse:
         :return: A HTTP response without the JWT token cookie
         """
         logout(request)
+
         redirect_url = settings.LOGIN_URL
-        response = redirect(reverse(redirect_url or "index"))
+        response = redirect(reverse(redirect_url))
         response.delete_cookie(settings.COLA_COOKIE_NAME)
         return response
 
 
-class IAIColaLogin(View):
+class ColaLogin(View):
```

Comment:
am open to what this should be called, but i think  we only need to say who we are once and not in ever class?